applications/MeshingApplication/custom_processes/metrics_hessian_process.h

```diff
@@ -187,6 +187,7 @@ class ComputeHessianSolMetricProcess
     double mMinSize;                         // The minimal size of the elements
     double mMaxSize;                         // The maximal size of the elements
     bool mEnforceCurrent;                    // With this we choose if we inforce the current nodal size (NODAL_H)
+    bool mEstimateInterpError;               // If the error of interpolation will be estimated
     double mInterpError;                     // The error of interpolation allowed
```

Comment:
I will not block the review for this, but i would have called this something like mTargetInterpolationError

Reply:
Ok